mne/tests/test_annotations.py

```diff
@@ -808,6 +808,10 @@ def dummy_annotation_file(tmpdir_factory, ch_names, fmt):
         content = ("onset,duration,description\n"
                    "2002-12-03 19:01:11.720100,1.0,AA\n"
                    "2002-12-03 19:01:20.720100,2.425,BB")
+    elif fmt == 'tsv':
+        content = ("onset\tduration\tdescription\n"
+                   "2002-12-03 19:01:11.720100\t1.0\tAA\n"
+                   "2002-12-03 19:01:20.720100\t2.425\tBB")
```

Comment:
this is not how BIDS events tsv files should be formatted

see for example

https://openneuro.org/datasets/ds000248/file-display/sub-01:meg:sub-01_task-audiovisual_run-01_events.tsv

Reply:
You mean `'    '` rather than `'\t'`?

Reply:
onset should be in seconds not time stamps
description should probably be called trial_type

Reply:
> onset should be in seconds not time stamps
description should probably be called trial_type

Yes re: the first comment.

re: the second point --> "trial_type" is an optional column so for the test it may make more sense here. "description" would fall under an "additional arbitrary, user-defined column", so it's absolutely fine, but would have to be described in events.json